mongo-fixes/neptune-migration/neptune-migration.sh

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
start scripts with a hashbang
```suggestion
#!/usr/bin/env bash
```